src/TimeNode/repl.js

```diff
@@ -62,10 +62,11 @@ const start = (timenode) => {
         const profit = bounties - costs
 
         const claimedPendingExecution = timenode.getClaimedNotExecutedTransactions()
+        const failedClaims = timenode.getUnsucessfullyClaimedTransactions()
 
         const stringToFixed = (string) => parseFloat(string).toFixed(6)
 
-        console.log(`${accountStats.account} | Executed: ${
+        console.log(`${accountStats.account} | Failed Claims: ${failedClaims.length} | Executed: ${
```

Comment:
This is becoming a long line. We should consider adding separating the log into multiple lines.